src/Sulu/Bundle/CategoryBundle/Resources/public/js/main.js

```diff
@@ -28,29 +28,29 @@ define({
 
         sandbox.mvc.routes.push({
             route: 'settings/categories',
-            callback: function(){
+            callback: function() {
                 return '<div data-aura-component="categories@sulucategory" data-aura-display="list"/>';
             }
         });
 
         sandbox.mvc.routes.push({
             route: 'settings/categories/new/:parent/:content',
-            callback: function(parent, content){
-                return '<div data-aura-component="categories/content@sulucategory" data-aura-display="form" data-aura-parent="'+ parent +'" data-aura-content="'+ content +'"/>';
+            callback: function(parent, content) {
```

Comment:
The ":content" par ist used by the tabs component. The whole tabs handling gets handled automatically, but we need to root the url the the component